src/mon/OSDMonitor.cc

```diff
@@ -6114,6 +6114,9 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 	wait_for_finished_proposal(op, new Monitor::C_Command(mon, op, 0, rs,
 						  get_last_committed() + 1));
 	return true;
+      } else {
+        ss << "can not create-or-move updated item name '" << name << "' weight " << weight
```

Comment:
what additional information is conveyed to user in addition to the parameters he/she sends to the mon?